frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/SingleLegalValue/SingleLegalValue.tsx

```diff
@@ -80,25 +80,28 @@ export const SingleLegalValue = ({
             <ConditionallyRender
                 condition={Boolean(legalValues.length)}
                 show={
-                    <FormControl component='fieldset'>
-                        <RadioGroup
-                            aria-label='selected-value'
-                            name='selected'
-                            value={value}
-                            onChange={(e) => {
-                                setError('');
-                                setValue(e.target.value);
-                            }}
-                        >
-                            {filteredValues.map((match) => (
-                                <LegalValueLabel
-                                    key={match.value}
-                                    legal={match}
-                                    control={<Radio />}
-                                />
-                            ))}
-                        </RadioGroup>
-                    </FormControl>
+                    <div style={{ display: 'flex' }}>
```

Comment:
what is the purpose of the extra div?

Reply:
It should encapsulate the fieldset. Intention was to limit the height when a lot of values so that it has a scroll.  This was an incomplete version